### PR TITLE
LPS-121920

### PR DESCRIPTION
**Pull request by jonmak08**
Hey @markocikos,

Attached is an update for http://issues.liferay.com/browse/LPS-121920.

Please let me know if you have any questions. Thanks!

### CODE REVIEW

File: modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {postForm} from 'frontend-js-web';
```
Comment (markocikos) [inline modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js:15]: Unused, we can remove this.

File: modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		}
+	};
+
+	const restoreSelectedEntries = () => {
+		const form = document.getElementById(`${portletNamespace}fm`);
+
+		if (form) {
+			submitForm(form);
+		}
+		submitForm(form, restoreEntriesURL);
```
Comment (markocikos) [inline modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js:42]: This does not look right, the entry is both deleted and restored? Probably needs to be
```js
if (form) {
    submitForm(form, restoreEntriesURL);
}
```

File: modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	};
+
+	return {
+		...otherProps,
+		onActionButtonClick: (event, {item}) => {
+			if (item?.data?.action === 'deleteSelectedEntries') {
```
Comment (markocikos) [inline modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js:48]: A minor SF,  we can put `item?.data?.action` in a `const`

**Review by markocikos — CHANGES_REQUESTED** [review]
Left a few comments inline, apart from few minor things, looks good!

When you are opening a PR, please include steps to test :pray: Even batter is additional screenshot or gif. This time I knew where `trash-web` is, but there are plenty of modules I or some other reviewer will not know. It's not rare for us to reach out to product teams to find out where the module is in UI.

If you are very unsure about the solution in PR, you can open it as a Draft on `liferay-frontend` fork. That way anybody can give broad-strokes feedback, not going in detail.

About this PR, please forward it with changes to `liferay-echo`.

**Comment by jonmak08** [comment]
Thanks for the feedback and the tips @markocikos, I appreciate it!  Have a good weekend.

**Comment by jonmak08** [comment]
New Pull Request: https://github.com/liferay-echo/liferay-portal/pull/3575

**Pull request closed without merging**